pkg/cloud/affinity_groups_test.go

```diff
@@ -49,8 +49,9 @@ var _ = Describe("AffinityGroup Unit Tests", func() {
 		fakeAG = &cloud.AffinityGroup{
 			Name: "FakeAffinityGroup",
 			Type: cloud.AffinityGroupType}
-		cluster = &infrav1.CloudStackCluster{Spec: infrav1.CloudStackClusterSpec{
-			Zone: "Zone1", Network: "SharedGuestNet1"}}
+		cluster = &infrav1.CloudStackCluster{
+			Spec:       infrav1.CloudStackClusterSpec{Zone: "Zone1", Network: "SharedGuestNet1"},
```

Comment:
I'm fine with this, but you should be able to skip the import and instead do `cluster.ObjectMeta.SetUID("0")`

Reply:
That sounds reasonable.  I'll do it.